reference/filter-reference.md

```diff
@@ -396,12 +393,11 @@ Settings that are only valid when `grantType: "AuthorizationCode"`:
 
 HTTP client settings for talking to the identity provider:
 
- - `maxStale`: How long to keep stale cached OIDC replies for. This sets the `max-stale` Cache-Control directive on requests, and also ignores the `no-store` and `no-cache` Cache-Control directives on responses.  This is useful for maintaining good performance when working with identity providers with mis-configured Cache-Control.
- - `insecureTLS` disables TLS verification when speaking to an identity provider with an `https://` `authorizationURL`.  This is
-   discouraged in favor of either using plain `http://` or [installing a self-signed certificate](#installing-self-signed-certificates).
+ - `maxStale`: How long to keep stale cached OIDC replies for. This sets the `max-stale` Cache-Control directive on requests, and also ignores the `no-store` and `no-cache` Cache-Control directives on responses.  This is useful for maintaining good performance when working with identity providers with misconfigured Cache-Control.
```

Comment:
Huh. I'd written it as "mis-configured" because my spellchecker complained about "misconfigured".